155.py

```diff
@@ -0,0 +1,34 @@
+class MinStack:
+    def __init__(self):
+        self.stack = []
+        self.min_stack = []
+        return
+
+    def push(self, val: int) -> None:
+        self.stack.append(val)
+        if len(self.min_stack) == 0:
+            self.min_stack.append(val)
+        else:
+            self.min_stack.append(min(self.min_stack[-1], val))
+        return
+
+    def pop(self) -> None:
+        self.stack.pop()
+        self.min_stack.pop()
+        return
```

Comment:
you do not need return statements at the end of `void` functions. Just leave it blank